node-process-argv/echo.ts

```diff
@@ -0,0 +1,2 @@
+const [, , ...args] = process.argv;
```

Comment:
You didn't have to account for multiple arguments but this definitely would have been the way to do that!